LocationManager/INTULocationManager/INTULocationManager.h

```diff
@@ -125,6 +130,33 @@ __INTU_ASSUME_NONNULL_BEGIN
 /** Immediately cancels the location request (or subscription) with the given requestID (if it exists), without executing the original request block. */
 - (void)cancelLocationRequest:(INTULocationRequestID)requestID;
 
+#pragma mark Heading Requests
+
+/** 
+ Whether or not the CLLocationManager is currently sending heading updates. 
+ */
+@property (nonatomic, assign, readonly) BOOL isUpdatingHeading;
```

Comment:
Is there any reason this is exposed as a public property? The corresponding one for location (`isUpdatingLocation`) is kept private, and I can't see a reason why this one should be any different.